Entity/ProductRepository.php

```diff
@@ -97,6 +97,23 @@ public function findByInternalItemNumber($internalItemNumber)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function findByGlobalTradeItemNumber($globalTradeItemNumber)
```

Comment:
this function is not needed since doctrine has magic methods that already do the same